x-pack/plugins/data_usage/kibana.jsonc

```diff
@@ -6,7 +6,7 @@
     "@elastic/security-solution"
   ],
   "group": "platform",
-  "visibility": "private",
+  "visibility": "shared",
```

Comment:
I'm making `@kbn/test-suites-serverless` part of _platform/shared_ in my [ongoing PR](https://github.com/elastic/kibana/pull/199630).
This should fix your warnings without having to make this package `shared`.

Are you okay rollbacking this change and ignoring warnings till my PR is merged?

Reply:
Yes, that sounds good.  Thank you!